packages/web/src/pages/sign-up-page/utils/selectArtistsPreviewContext.tsx

```diff
@@ -41,64 +43,73 @@ export const SelectArtistsPreviewContextProvider = (props: {
   const { data: artistTracks } = useGetUserTracksByHandle(
     {
       handle: artist?.handle,
-      currentUserId: null
+      currentUserId: null,
+      // Unlikely we cant play an artist's first 3 tracks.
+      limit: 3
     },
     { disabled: !artist?.handle }
   )
 
   useEffect(() => {
-    const trackId = artistTracks?.find((track) => track.is_available)?.track_id
-    trackId && setTrackId(encodeHashId(trackId))
+    const track = artistTracks?.find((track) => track.is_available)
+    if (track) {
+      setTrack(track)
+    }
   }, [artistTracks])
 
   const togglePlayback = useCallback(() => {
     if (audioPlayer.isPlaying()) {
-      audioPlayer.pause()
+      dispatch(playerActions.pause())
```

Comment:
interesting, for my own learning; what was the difference here?

Reply:
for pause i don't know how much it matters, but i wanted to make sure the redux store stayed in sync with the common user actions rather than having play go through redux, and pause go through audioplayer directly